### PR TITLE
Add "wcaid" phrase

### PR DESCRIPTION
**Pull request by leduyquang753**
**What I changed:**
* Added new "wcaid" phrase.
* Made the **Competitor** field longer.

I hope this could be merged in the next 0.14.0 release. Thank you.
**This pull request is outdated and will be closed.**

### CODE REVIEW

File: webscrambles/src/net/gnehzr/tnoodle/server/webscrambles/ScrambleRequest.java [hunk 1]
```diff
@@ -102,8 +102,8 @@
     private static BaseFont monoFont, sansSerifFont;
     static {
         try {
-            monoFont = BaseFont.createFont("fonts/LiberationMono-Regular.ttf", BaseFont.IDENTITY_H, BaseFont.EMBEDDED);
-            sansSerifFont = BaseFont.createFont("fonts/wqy-microhei.ttf", BaseFont.IDENTITY_H, BaseFont.EMBEDDED);
+            monoFont = BaseFont.createFont("fonts/consola.ttf", BaseFont.IDENTITY_H, BaseFont.EMBEDDED);
+            sansSerifFont = BaseFont.createFont("fonts/segoeui.ttf", BaseFont.IDENTITY_H, BaseFont.EMBEDDED);
```
Comment (jfly) [inline webscrambles/src/net/gnehzr/tnoodle/server/webscrambles/ScrambleRequest.java]: Opps?

File: webscrambles/src/i18n/en.yml [hunk 2]
```diff
@@ -9,6 +9,7 @@ en:
         event: Fewest Moves
         graded: Graded by
         result: Result
+        wcaid: WCA ID
```
Comment (jfly) [inline webscrambles/src/i18n/en.yml]: nit: I think `wcaid` should be `wca_id`

**Comment by leduyquang753** [comment]
Looks like my commits are between other updating commits **(and somehow I am considered as the co-author of the fr update?)** I'm trying to squash my commits via **git rebase** but the other commits are just keep showing off...
And the desired length of the **Competitor** line is shown in #301.
Could you tell me how to squash those commits without affecting the others? Thanks.

**Comment by jfly** [comment]
Currently you've got conflict markers committed (take a look at your diff here: https://github.com/thewca/tnoodle/pull/302/files). I think the simplest fix for you would be to reset your branch to match thewca `master` branch, and manually re-apply your changes (since it's only a few lines of code changed).

**Comment by leduyquang753** [comment]
I'm going to close this to open another pull request, which improves the form lines (making them gray so that they are easier to write)...

**Pull request closed without merging**